### DISCUSSION TITLE
Difficulty ranges for multiplayer lobbies

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#24912] [Ideas] [answered]
The idea is to add to the settings of multiplayer lobbies a difficulty range that lets the host set a star range and any maps that fall outside that range (too easy or too difficult) cannot be added by anyone, excluding perhaps the host. It might make sense to still allow the host to add whatever difficulty they want, just not anyone else.

**Comment by peppy** [comment, marked as answer]
Definitely on the short list of things which need to be implemented for multiplayer.

**Comment by huyenden** [comment]
Agree with this topic, to avoid the situation where players just add beatmaps beyond the allowed distance as a way to troll, exit the room without playing, causing the room owner to have a headache deleting those beatmaps.

**Reply by ghost** [reply]
> Agree with this topic, to avoid the situation where players just add beatmaps beyond the allowed distance as a way to troll, exit the room without playing, causing the room owner to have a headache deleting those beatmaps.

I think that's almost a separate issue because you can still add trash maps that are within the difficulty range. There should be a way for the host to clear the queue if it takes long to clear manually.

**Reply by huyenden** [reply]
If within the difficulty range, maps that make players uncomfortable such as maps ranked ar0 4*, ranked cs7 5* are still acceptable. As for the Graveyard map, I don't know. But limiting the beatmap beyond the allowed difficulty as you mentioned would still be better
